src/mnode/mnode-governance.cpp

```diff
@@ -121,12 +121,38 @@ CAmount CMasternodeGovernance::IncrementTicketPaidAmount(CAmount payment, CGover
     return aAmountPaid;
 }
 
+bool CMasternodeGovernance::CanVote(std::string& strErrorRet) const
```

Comment:
can make it noexcept...
IsActiveMasterNode,IsEnabled,IsBroadcastedWithin - noexcept
Just make also CMasternodeMan::Get const noexcept- it does not throw any exceptions